.dockerignore

```diff
@@ -0,0 +1,6 @@
+sandbox/db-data
+tsc
+venv
+images
+ci
+samples
```

Comment:
nit: It would be good to alphabetize this list, if we can.

Reply:
Done